src/components/calculator.js

```diff
@@ -1,71 +1,68 @@
-          total={total}
-          next={next}
-          operation={operation}
-        />
-        <span className="d-flex row">
-          <button type="button" data="AC">AC</button>
-          <button type="button" data="+/-">
-            +
-            <small>/</small>
-            -
-          </button>
-          <button type="button" data="%">%</button>
-          <button type="button" data="/" className="operator">&divide;</button>
-        </span>
-        <span className="d-flex row">
-          <button type="button" data="7">7</button>
-          <button type="button" data="8">8</button>
-          <button type="button" data="9">9</button>
-          <button type="button" data="x" className="operator">&times;</button>
-        </span>
-        <span className="d-flex row">
-          <button type="button" data="4">4</button>
-          <button type="button" data="5">5</button>
-          <button type="button" data="6">6</button>
-          <button type="button" data="-" className="operator">-</button>
-        </span>
-        <span className="d-flex row">
-          <button type="button" data="1">1</button>
-          <button type="button" data="2">2</button>
-          <button type="button" data="3">3</button>
-          <button type="button" data="+" className="operator">+</button>
-        </span>
-        <span className="d-flex row">
-          <button type="button" data="0" className="zero">0</button>
-          <button type="button" data=".">.</button>
-          <button type="button" data="=" className="operator">=</button>
-        </span>
-      </div>
-    );
-  }
-}
+  const { total, next, operation } = state;
+  return (
+    <div className="calculator cols">
+      <Answer
+        total={total}
+        next={next}
+        operation={operation}
+      />
+      <span className="d-flex row">
+        <button type="button" data="AC">AC</button>
+        <button type="button" data="+/-">
+          +
+          <small>/</small>
+          -
+        </button>
+        <button type="button" data="%">%</button>
+        <button type="button" data="÷" className="operator">&divide;</button>
+      </span>
+      <span className="d-flex row">
+        <button type="button" data="7">7</button>
+        <button type="button" data="8">8</button>
+        <button type="button" data="9">9</button>
+        <button type="button" data="x" className="operator">&times;</button>
+      </span>
+      <span className="d-flex row">
+        <button type="button" data="4">4</button>
+        <button type="button" data="5">5</button>
+        <button type="button" data="6">6</button>
+        <button type="button" data="-" className="operator">-</button>
+      </span>
+      <span className="d-flex row">
+        <button type="button" data="1">1</button>
+        <button type="button" data="2">2</button>
+        <button type="button" data="3">3</button>
+        <button type="button" data="+" className="operator">+</button>
```

Comment:
- [ ] [OPTIONAL] For your data you may need to create an array and loop through it
- It may look like something like this :point_down: 
![image](https://user-images.githubusercontent.com/75126481/143580273-542ffa6f-ac38-4f3d-96d8-f929af8aeefe.png)

Reply:
For this solution, I was avoiding too many loops in the program. I choose to use one loop to dynamically get the data, which works perfectly encase of future changes.